packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.controller.js

```diff
@@ -0,0 +1,64 @@
+import { NSX_RESOURCES } from './dedicatedCloud-datacenter-manage-nsx.constants';
+import { EDGES_SIZES } from '../../../datacenters/datacenter.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(DedicatedCloud, $translate, coreURLBuilder) {
+    this.DedicatedCloud = DedicatedCloud;
+    this.$translate = $translate;
+    this.coreURLBuilder = coreURLBuilder;
+    this.NSX_RESOURCES = NSX_RESOURCES;
+  }
+
+  $onInit() {
+    this.loading = false;
+    this.nsx_sizes = Object.keys(EDGES_SIZES);
+    this.selectedNsxLevel = null;
+    this.hasScalingCapabilities =
+      this.nsxtEdgesScalingCapabilities.length ===
+      Object.keys(EDGES_SIZES).length;
+  }
+
+  getDatastoreOrderLink() {
+    return this.coreURLBuilder.buildURL(
+      'dedicated',
+      `#/dedicated_cloud/${this.serviceName}/datacenter/${this.datacenterId}/datastores/order`,
+    );
+  }
+
+  getHostOrderLink() {
+    return this.coreURLBuilder.buildURL(
+      'dedicated',
+      `#/dedicated_cloud/${this.serviceName}/datacenter/${this.datacenterId}/hosts/order`,
+    );
+  }
```

Comment:
for getDatastoreOrderLink() and getHostOrderLink() maybe use constants (in routing for exemple) rather than functions because these are static and not dynamic things that only have to be generated once.

Reply:
Good point, I add the links in the routing